integrations/opentelemetry/src/main/kotlin/com/xebia/functional/xef/opentelemetry/OpenTelemetryCounter.kt

```diff
@@ -0,0 +1,14 @@
+package com.xebia.functional.xef.opentelemetry
+
+import com.xebia.functional.xef.metrics.CounterMetric
+
+class OpenTelemetryCounter(private val longCounter: io.opentelemetry.api.metrics.LongCounter) :
+  CounterMetric {
+  override fun increment(n: Long) {
+    longCounter.add(n)
+  }
+
+  override fun decrement(n: Long) {
+    longCounter.add(n)
+  }
```

Comment:
Why do we call the' add' method if the function's purpose is decrementing the counter?

Reply:
Removed, thanks!